src/bootstrap/bootstrap.py

```diff
@@ -315,7 +315,7 @@ def build_triple(self):
         try:
             ostype = subprocess.check_output(['uname', '-s']).strip().decode(default_encoding)
             cputype = subprocess.check_output(['uname', '-m']).strip().decode(default_encoding)
-        except (subprocess.CalledProcessError, WindowsError):
+        except subprocess.CalledProcessError:
```

Comment:
WindowsError is just a [subclass of OSError](https://docs.python.org/2/library/exceptions.html), so that would be the portable equivalent:

> exception WindowsError
>     Raised when a Windows-specific error occurs or when the error number does not correspond to an errno value. The winerror and strerror values are created from the return values of the GetLastError() and FormatMessage() functions from the Windows Platform API. The errno value maps the winerror value to corresponding errno.h values. This is a subclass of OSError.

So I'll just change it to use that, avoiding the portability issues entirely.